modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/delegate/ConfigModelLoaderDelegate.java

```diff
@@ -51,12 +51,10 @@ void declareConfigurations(ExtensionDeclarer declarer, ExtensionModelParser exte
       configParser.getDisplayModel().ifPresent(d -> configurationDeclarer.getDeclaration().setDisplayModel(d));
       configParser.getExternalLibraryModels().forEach(configurationDeclarer::withExternalLibrary);
       configParser.getAdditionalModelProperties().forEach(configurationDeclarer::withModelProperty);
-      if (configParser.mustResolveMinMuleVersion()) {
-        configParser.getResolvedMinMuleVersion().ifPresent(resolvedMMV -> {
-          configurationDeclarer.withMinMuleVersion(resolvedMMV.getMinMuleVersion());
-          LOGGER.debug(resolvedMMV.getReason());
-        });
-      }
+      configParser.getResolvedMinMuleVersion().ifPresent(resolvedMMV -> {
+        configurationDeclarer.withMinMuleVersion(resolvedMMV.getMinMuleVersion());
+        LOGGER.debug(resolvedMMV.getReason());
```

Comment:
not very useful. At the very least, it would need information about which element is this for.

Reply:
that is already available on the data passed to the `ResolvedMinMuleVersion` constructor